### ISSUE TITLE
Use setuptools again

### ISSUE THREAD

**Issue by kbabioch** [sdispater/pendulum#189]
I'm packaging this software in [AUR](https://aur.archlinux.org/packages/python-pendulum/) for Arch Linux. Support for setuptools was dropped with commit 3c0bdcbabce38e4e809fb9ccc7a12e3210eadead, in favour of poetry. While I can understand that you want to push your own package standard, I can attest to the fact that packaging your software differently than anyone else, is a headache for distribution packagers. Basically I cannot package your software anymore without getting into `poetry`.

In the poetry README you state:

    The package is highly experimental at the moment so expect things to change and break.

That sound's crazy to me. Isn't it possible to support both of them in parallel?

**Comment by sdispater** [comment]
I think by `setuptools` you mean `setup.py`, since `setuptools` is not part of the standard library and, as such, is not required to install packages.

I understand that it makes things a bit harder for you. But there is a way to retrieve the old format with `poetry`:

```bash
poetry build --format sdist
```

You will get a `.tar.gz` distribution which contains the `setup.py` file. It should help with the transition.

And bear in mind that the `setup.py` file will no longer be required in the future with the new `pyproject.toml` file standard (see [PEP 517](https://www.python.org/dev/peps/pep-0517/) and [PEP 518](https://www.python.org/dev/peps/pep-0518/). So the assumption about `setup.py` will no longer be true and you will have to use whatever tool the package as declared as the build system.
